client/.travis.yml

```diff
@@ -0,0 +1,23 @@
+---
+language: node_js
+node_js:
+  - "0.12"
```

Comment:
Yeah, when they merged io.js in they bumped up to 4.0 because io.js was at release 3. They also committed to Long Term Support releases that are supposed to get bug and security fixes for 18 months or something like that. So 5.x is latest stable, but 4.x is the current LTS release.